packages/ember-runtime/lib/mixins/array.js

```diff
@@ -1237,126 +1233,20 @@ const ArrayMixin = Mixin.create(Enumerable, {
     @public
   */
   '@each': computed(function() {
-    // TODO use Symbol or add to meta
-    if (!this.__each) {
-      this.__each = new EachProxy(this);
-    }
+    deprecate(
+      "Getting the '@each' property is deprecated",
```

Comment:
The message needs to include more details, or it’s going to be very hard to track down...